docs/7 Frontend/1 Overview.md

```diff
@@ -0,0 +1,19 @@
+# Frontend Overview
+
+When scaffolding a blockchain, Starport also creates a Vue frontend application in `vue` directory. If this directory exists, `starport serve` automatically runs `npm run serve`.
```

Comment:
```suggestion
A Vue frontend app is created in `vue` directory when a blockchain is scaffolded. If the `vue` directory exists, the `starport serve` command automatically runs `npm run serve`.
```

Reply:
@fadeev what  `npm run serve` do? why only for existing directories?

Reply:
`npm run serve` runs the frontend. We execute this command only if `vue` exists, otherwise there is nothing to run on the frontend.